src/Cake.Common/Tools/OctopusDeploy/PushPackageSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cake.Common.Tools.OctopusDeploy
+{
+    /// <summary>
+    /// Contains settings used by <see cref="OctopusDeployPusher.PushPackage"/>.
+    /// </summary>
+    public sealed class PushPackageSettings : OctopusDeploySettings
```

Comment:
@agc93 I think this class should be called `OctopusPushSettings` to be consistent with other classes in this section of the code.